README.md

```diff
@@ -25,6 +25,8 @@ If you want to use this bundle with previous Symfony versions, please use 0.2.x
 
 **It's important you manually require either Doctrine's ORM or MongoDB ODM as well, these packages are not required automatically as you can choose between them. Failing to do so may trigger errors on installation**
 
+If using Symfony 5.2 or older, you will also need to install the `symfony/security-guard` package, it is only required for the legacy authentication API and is not compatible with Symfony 6.0.
```

Comment:
The Guard component is deprecated and dropped in Symfony 6.  I made it an optional dependency, folks on Symfony 4.4 will need to manually install that component to keep using this package.  Folks on Symfony 5.3 and later won't need to have it installed since the newer API doesn't use it.